src/Symfony/Component/Messenger/Transport/Serialization/Serializer.php

```diff
@@ -40,7 +41,7 @@ public function __construct(SymfonySerializerInterface $serializer = null, strin
     {
         $this->serializer = $serializer ?? self::create()->serializer;
         $this->format = $format;
-        $this->context = $context;
+        $this->context = $context + [self::MESSENGER_SERIALIZATION_CONTEXT => true];
```

Comment:
I see that you changed your mind about this, but I would revert this. We should support normalization for all FlattenException instances.

Reply:
I explained the why here: https://github.com/symfony/symfony/pull/37087#issuecomment-678079420